clients/admin-ui/src/features/config-wizard/AuthenticateOktaForm.tsx

```diff
@@ -0,0 +1,214 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionItem,
+  AccordionPanel,
+  Button,
+  Divider,
+  Heading,
+  HStack,
+  Stack,
+  Text,
+} from "@fidesui/react";
+import { Form, Formik } from "formik";
+import { useState } from "react";
+import * as Yup from "yup";
+
+import { useAppDispatch, useAppSelector } from "~/app/hooks";
+import DocsLink from "~/features/common/DocsLink";
+import { CustomTextInput } from "~/features/common/form/inputs";
+import {
+  isErrorResult,
+  ParsedError,
+  parseError,
+} from "~/features/common/helpers";
+import { useAlert } from "~/features/common/hooks";
+import {
+  Dataset,
+  GenerateResponse,
+  GenerateTypes,
+  System,
+  ValidTargets,
+} from "~/types/api";
+import { RTKErrorResult } from "~/types/errors";
+
+import {
+  changeStep,
+  selectOrganizationFidesKey,
+  setSystemsForReview,
+} from "./config-wizard.slice";
+import { DOCS_URL_OKTA_TOKEN } from "./constants";
+import { useGenerateMutation } from "./scanner.slice";
+import ScannerError from "./ScannerError";
+import ScannerLoading from "./ScannerLoading";
+
+const isSystem = (sd: System | Dataset): sd is System => "system_type" in sd;
+
+const initialValues = {
+  orgUrl: "",
+  token: "",
+};
+
+type FormValues = typeof initialValues;
+
+const ValidationSchema = Yup.object().shape({
+  orgUrl: Yup.string()
```

Comment:
Yup has built in url validation: https://www.npmjs.com/package/yup#stringurlmessage-string--function-schema